js/ui/drop_down_box.js

```diff
@@ -213,6 +217,25 @@ const DropDownBox = DropDownEditor.inherit({
         return this.callBase();
     },
 
+    ///#DEBUG
```

Comment:
My suggestion is to export `realDevice` object from the current module:

```
///#DEBUG
module.exports.realDevice = realDevice;
```
and use it directly in the test.

So, we can avoid adding a new method to the component.